### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by artnorton**
## Status
Ready/In Progress/In Hold(Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------

## Required version of Demisto
x.x.x

## Does it break backward compatibility?
   - Yes
       - Further details:
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Dependencies
Mention the dependencies of the entity you changed as given from the precommit hooks in checkboxes, and tick after tested them.
- [ ] Dependency 1
- [ ] Dependency 2
- [ ] Dependency 3

## Additional changes
Describe additional changes done, for example adding a function to common server.

### CODE REVIEW

File: Integrations/integration-Screenshot_Machine.yml [hunk 1]
```diff
@@ -0,0 +1,138 @@
+commonfields:
+  id: Screenshot Machine_copy
```
Comment (Itay4) [inline Integrations/integration-Screenshot_Machine.yml]: remove `_copy`

File: Integrations/integration-Screenshot_Machine.yml [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
...QAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAIUoAAFKEABClCAAhSgAAUoQAEKUIACFKAABShAAQpQgAKdCfwfc1TwvPFzSvkAAAAASUVORK5CYII=
```
Comment (Itay4) [inline Integrations/integration-Screenshot_Machine.yml]: image size is not according to our standard, please get one from eynav

File: Integrations/integration-Screenshot_Machine.yml [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+description: Uses screenshot machine to get a screenshot
+configuration:
+- display: Secret Phrase
+  name: secret
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: ""
```
Comment (Itay4) [inline Integrations/integration-Screenshot_Machine.yml]: missing display name

File: Integrations/integration-Screenshot_Machine.yml [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+description: Uses screenshot machine to get a screenshot
+configuration:
+- display: Secret Phrase
```
Comment (Itay4) [inline Integrations/integration-Screenshot_Machine.yml]: need to add detailed description how to get the credentials

File: Integrations/integration-Screenshot_Machine.yml [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+        getURL = "http://api.screenshotmachine.com/?key=861542&url=" + url + "&dimension=" + argDict.get('dimension') + "&device=" + argDict.get('device') + "&format=jpg&hash=" + md5Hash + "&cacheLimit=" + argDict.get('cacheLimit') + "&delay=" + argDict.get('delay')
+        screenshot=site_lookup(getURL)
+        #screenshot = requests.get(a)
+        demisto.results(screenshot)
+
+        # EXECUTION
+    LOG('command is %s' % (demisto.command(), ))
+    try:
+        if demisto.command() == 'test-module':
+            test()
```
Comment (Itay4) [inline Integrations/integration-Screenshot_Machine.yml]: there isn't such function

File: Integrations/integration-Screenshot_Machine.yml [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+
+    except Exception, e:
+        LOG(e.message)
+        LOG.print_log()
+        raise
+
+
+  type: python
+  commands:
+  - name: screenshot
```
Comment (Itay4) [inline Integrations/integration-Screenshot_Machine.yml]: please improve command name

File: Integrations/integration-Screenshot_Machine.yml [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+        #screenshot = requests.get(a)
+        demisto.results(screenshot)
+
+        # EXECUTION
+    LOG('command is %s' % (demisto.command(), ))
+    try:
+        if demisto.command() == 'test-module':
+            test()
+
+        if demisto.command() == 'screenshot':
```
Comment (Itay4) [inline Integrations/integration-Screenshot_Machine.yml]: use `elif`

File: Integrations/integration-Screenshot_Machine.yml [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+
+    def get_screenshot():
+        url = demisto.args()['url']
+        device = demisto.args()['device']
+        dimension = demisto.args()['dimension']
+        cacheLimit = demisto.args()['cacheLimit']
+        delay = demisto.args()['delay']
+        md5Hash = generateHash(url, "Henry Phipps Jr.")
+        argDict = demisto.args()
+        getURL = "http://api.screenshotmachine.com/?key=861542&url=" + url + "&dimension=" + argDict.get('dimension') + "&device=" + argDict.get('device') + "&format=jpg&hash=" + md5Hash + "&cacheLimit=" + argDict.get('cacheLimit') + "&delay=" + argDict.get('delay')
```
Comment (Itay4) [inline Integrations/integration-Screenshot_Machine.yml]: why `http` ?

File: Integrations/integration-Screenshot_Machine.yml [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+    API_KEY = demisto.params().get('apikey')
+
+    def site_lookup(a):
+
+        r = requests.get(a, allow_redirects=True)
+        i = BytesIO(r.content)
+        res = fileResult('myfile',i.read())
+
+        if r.status_code < 200 or r.status_code >= 300:
+            return_error_and_exit('Failed to update Content.\nURL: {}, Status Code: {}, Response: {}'.format(a, r.status_code, r.text))
```
Comment (Itay4) [inline Integrations/integration-Screenshot_Machine.yml]: there isnt such function `return_error_and_exit`, use `return_error`

File: Integrations/integration-Screenshot_Machine.yml [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+
+        return hashlib.md5(url + secretKey).hexdigest()
+
+    def get_screenshot():
+        url = demisto.args()['url']
+        device = demisto.args()['device']
+        dimension = demisto.args()['dimension']
+        cacheLimit = demisto.args()['cacheLimit']
+        delay = demisto.args()['delay']
+        md5Hash = generateHash(url, "Henry Phipps Jr.")
```
Comment (Itay4) [inline Integrations/integration-Screenshot_Machine.yml]: whats `"Henry Phipps Jr."` ?

**Comment by yardensade** [comment]
What is the status here? this is stale for about 4 months

**Comment by Itay4** [comment]
closing and uploading yml to issue

**Pull request closed without merging**